boards/ti/lp_am243x/lp_am243x_am2434_r5f0_0.dts

```diff
@@ -0,0 +1,172 @@
+		led0 = &test_led1_green;
+		adc0 = &adc0;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+
+		test_led1_green: led_1 {
+			gpios = <&gpio0 22 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	rsc_table: memory@a0100000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0xa0100000 DT_SIZE_K(4)>;
+		zephyr,memory-region = "RSC_TABLE";
+		zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_RAM) )>;
+	};
+
+	ipc_shm0: memory@a5000000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0xa5000000 DT_SIZE_M(8)>;
+		zephyr,memory-region = "IPC_SHM";
+		zephyr,memory-attr = <( DT_MEM_ARM(ATTR_MPU_IO) )>;
+	};
+
+	ipc0: ipc {
+		compatible = "zephyr,mbox-ipm";
+		mboxes = <&mbox6 0>, <&mbox6 1>;
+		mbox-names = "rx", "tx";
+	};
+};
+
+/* system timer */
+&timer8 {
+	status = "okay";
+};
+
+&uart0 {
+	status = "okay";
+	pinctrl-0 = <&uart0_tx &uart0_rx>;
+	pinctrl-names = "default";
+	current-speed = <115200>;
+};
+
+&gpio0 {
+	pinctrl-0 = <&gpio0_led1>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&i2c0 {
+	pinctrl-0 = <&i2c0_scl &i2c0_sda>;
+	pinctrl-names = "default";
+	status = "okay";
+};
+
+&adc0 {
+	ti,vrefp = <1800>;
+	ti,fifo = <0>;
+	status = "okay";
+
+	channel@0 {
+		reg = <0>;
+		ti,open-delay = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <0>;
+		zephyr,resolution = <12>;
+		zephyr,oversampling = <4>;
+	};
+
+	channel@1 {
+		reg = <1>;
+		ti,open-delay = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <0>;
+		zephyr,resolution = <12>;
+		zephyr,oversampling = <4>;
+	};
+
+	channel@2 {
+		reg = <2>;
+		ti,open-delay = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <0>;
+		zephyr,resolution = <12>;
+		zephyr,oversampling = <4>;
+	};
+
+	channel@3 {
+		reg = <3>;
+		ti,open-delay = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <0>;
+		zephyr,resolution = <12>;
+		zephyr,oversampling = <4>;
+	};
+
+	channel@4 {
+		reg = <4>;
+		ti,open-delay = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <0>;
+		zephyr,resolution = <12>;
+		zephyr,oversampling = <4>;
+	};
+
+	channel@5 {
+		reg = <5>;
+		ti,open-delay = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <0>;
+		zephyr,resolution = <12>;
+		zephyr,oversampling = <4>;
+	};
+
+	channel@6 {
+		reg = <6>;
+		ti,open-delay = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <0>;
+		zephyr,resolution = <12>;
+		zephyr,oversampling = <4>;
+	};
+
+	channel@7 {
+		reg = <7>;
+		ti,open-delay = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_INTERNAL";
+		zephyr,acquisition-time = <0>;
+		zephyr,resolution = <12>;
+		zephyr,oversampling = <4>;
+	};
+};
```

Comment:
These should not be here, I made a PR to change that for am243x evm as well: https://github.com/zephyrproject-rtos/zephyr/pull/92170